_arch/ficam_standards_policies.md

```diff
@@ -85,54 +85,54 @@ Each section of this page lists documents in reverse chronological order, with t
 
-<a href="https://csrc.nist.gov/publications/detail/sp/800-162/final" target="_blank" rel="noopener noreferrer">**NIST SP 800-162: Guide to Attribute Based Access Control (ABAC) Definition and Considerations**</a> (PDF, January 2014)<br/>
+<a href="https://nvlpubs.nist.gov/nistpubs/specialpublications/NIST.SP.800-162.pdf" target="_blank" rel="noopener noreferrer">**NIST SP 800-162: Guide to Attribute Based Access Control (ABAC) Definition and Considerations**</a> (PDF, January 2014)<br/>
 > This guideline provides federal agencies with a definition of ABAC. ABAC is a logical access control methodology in which authorization to perform a set of operations is determined by evaluating attributes associated with the subject, object, requested operations, and, in some cases, environment conditions against policy, rules, or relationships that describe the allowable operations for a given set of attributes.
 
-<a href="https://csrc.nist.gov/publications/detail/fips/201/2/final" target="_blank" rel="noopener noreferrer">**FIPS 201-2: PIV of Federal Employees and Contractors**</a> (PDF, August 2013)<br/>
+<a href="https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.201-3.pdf" target="_blank" rel="noopener noreferrer">**FIPS 201-3: PIV of Federal Employees and Contractors**</a> (PDF, August 2013)<br/>
 > This standard specifies the architecture and technical requirements for a common identification standard for federal employees and contractors. The overall goal is to achieve appropriate security assurance for multiple applications by efficiently verifying the claimed identity of individuals seeking physical access to federally controlled government facilities and electronic access to government information systems.
 
-<a href="https://csrc.nist.gov/publications/detail/sp/800-76/2/final" target="_blank" rel="noopener noreferrer">**NIST SP 800-76-2: Biometric Data Specification for PIV**</a> (PDF, July 2013)<br/>
+<a href="https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-76-2.pdf" target="_blank" rel="noopener noreferrer">**NIST SP 800-76-2: Biometric Data Specification for PIV**</a> (PDF, July 2013)<br/>
 > This guideline contains technical specifications for biometric data mandated in FIPS. These specifications reflect the design goals of interoperability and performance of the PIV credential. This specification addresses image acquisition to support the background check, fingerprint template creation, retention, and authentication. The biometric data specification in this document is the mandatory format for biometric data carried in the PIV Data Model (SP 800-73-1, Appendix A). Biometric data used only outside the PIV Data Model is not within the scope of this standard.
 
-<a href="https://csrc.nist.gov/publications/detail/sp/800-122/final" target="_blank" rel="noopener noreferrer">**NIST SP 800-122: Guide for Protecting the Confidentiality of Personally Identifiable Information (PII)**</a> (PDF. April 2010)<br/>
```

Comment:
@claytonjbarnette why did you update these to point to the PDF instead of the webpage?

Reply:
Thanks, Clayton. We link to the page with the doc rather than the actual doc so we don't have to keep track of when documents are updated. Please change these NIST references back to link to the page rather than the document.